fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgres/PostgresResourceDAO.java

```diff
@@ -76,7 +74,7 @@ public PostgresResourceDAO(Connection connection, String schemaName, FHIRDbFlavo
      * @throws FHIRPersistenceException
      */
     @Override
-    public Resource insert(Resource resource, List<ExtractedParameterValue> parameters, ParameterDAO parameterDao)
+    public Resource insert(Resource resource, List<ExtractedParameterValue> parameters, String parameterHashB64, ParameterDAO parameterDao)
```

Comment:
Add a @param tag for parameterHashB64

Reply:
It's an override, so I removed the comment entirely